nilearn/interfaces/fmriprep/tests/test_load_confounds.py

```diff
@@ -257,31 +312,42 @@ def test_load_single_confounds_file(tmp_path, fmriprep_version):
     """Check that the load_confounds function returns the same confounds \
     as _load_single_confounds_file."""
     nii_file, confounds_file = create_tmp_filepath(
-        tmp_path,
-        copy_confounds=True,
-        fmriprep_version=fmriprep_version)
+        tmp_path, copy_confounds=True, fmriprep_version=fmriprep_version
+    )
 
     # get defaults from load_confounds
     import inspect
-    _defaults = {key: value.default for key, value
-                 in inspect.signature(load_confounds).parameters.items()}
+
+    _defaults = {
+        key: value.default
+        for key, value in inspect.signature(load_confounds).parameters.items()
+    }
     _defaults.pop("img_files")
     _default_strategy = _defaults.pop("strategy")
 
-    _, confounds = _load_single_confounds_file(str(confounds_file),
-                                               strategy=_default_strategy,
-                                               **_defaults)
-    confounds_nii, _ = load_confounds(nii_file,
-                                      strategy=_default_strategy,
-                                      **_defaults)
+    _, confounds = _load_single_confounds_file(
+        str(confounds_file), strategy=_default_strategy, **_defaults
+    )
+    confounds_nii, _ = load_confounds(
+        nii_file, strategy=_default_strategy, **_defaults
+    )
     pd.testing.assert_frame_equal(confounds, confounds_nii)
 
 
-@pytest.mark.parametrize("strategy,message",
-                         [(["string", ], "not a supported type of confounds."),
-                          ("error", "tuple or list of strings"),
-                          ((0, ), "not a supported type of confounds."),
-                          (("compcor", ), "high_pass")])
+@pytest.mark.parametrize(
+    "strategy,message",
+    [
+        (
+            [
+                "string",
+            ],
+            "not a supported type of confounds.",
```

Comment:
this one is particularly ugly

Reply:
Agreed.

I removed some trailing comma in a a few places to force a reformating on one line. Still not great but at least less of an eyesore.